### DISCUSSION TITLE
Commits didn't show up

### DISCUSSION THREAD

**Discussion by Jean-Pierre-97s** [community/community#65367] [Repositories] [answered]
### Select Topic Area

Bug

### Body

Enviei alguns commits ontem para o github mas por algum motivo eles não estão aparecendo nas contribuições na minha conta.I uploaded some commits yesterday to github but for some reason they are not showing up in the contributions in my account.

**Comment by arafajs** [comment]
- The commits were made less than 24 hours ago. GitHub contributions are updated once a day, so it may take up to 24 hours for your new commits to show up.
- The email address used for the commit is not connected to your GitHub account. Commits must be made with an email address that is connected to your GitHub account in order to appear in your contributions graph. 
- The commits were made to a private repository. Contributions are only tracked for public repositories. If you made your commits to a private repository, they will not show up in your contributions.
- There is a caching issue. Sometimes, GitHub's contribution tracking system can get out of sync. If you're sure that you made the commits and they are not showing up in your contributions, you can try clearing your GitHub cache. To do this, follow these steps:

1. Go to your GitHub profile.
2. Click on the Settings tab.
3. Click on the Developer settings tab.
4. Scroll down to the Cache section and click on the Clear cache button.

**Reply by Jean-Pierre-97s** [reply]
I will try to wait the 24 hours, but i tried to follow the steps to the clear cache option and i didnt find it

**Comment by IceTheCoder** [comment, marked as answer]
If the commits were on a private repo, [this](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile) should help.

Also, make sure the branch you made the commits on has been published (so it's not just local, you can see it online on your repo's page) AND merged into main.

**Reply by Jean-Pierre-97s** [reply]
They only show in the contributions when they are on the main branch ?

**Reply by IceTheCoder** [reply]
I think. I once lost my GitHub streak after committing to another branch and thinking it'd count as a contribution. I'm not sure if the branch I committed to was local or published.

**Reply by Jean-Pierre-97s** [reply]
Thank you, man. All my commits are published but in another branch. When I merge them into the main branch, they all appear in the contribution area correctly.

**Reply by IceTheCoder** [reply]
Glad to be of help.

**Comment by Arkenstine** [comment]
Were they in a public repository?

**Reply by Jean-Pierre-97s** [reply]
Yes, they are in a public repository, I noticed that when I send them to main or make a pull request they start counting, but in other branches like develop they don't appear
